src/components/UI/Text/InputNumber.vue

```diff
@@ -155,7 +160,7 @@ export default {
         return this.value;
       },
       set(localValue) {
-        this.$emit("update:value", localValue);
+       this.$emit("update:value", localValue);
```

Comment:
Always perform a self review before giving a pr to someone else for review. 
All you have to do is go to the "files changed" tab and see your changes yourself. 
You'll figure out these small small issues on your own.
Like here, this removal of space is not necessary here.

Reply:
okay